pkg/resources/aws/aws_test.go

```diff
@@ -196,3 +198,67 @@ func TestSharedVolume(t *testing.T) {
 		}
 	}
 }
+
+func TestMatchesElbTags(t *testing.T) {
+	tc := []struct {
+		tags     map[string]string
+		actual   []*elb.Tag
+		expected bool
+	}{
+		{
+			tags: strtomap("tagkey1", "tagvalue1"),
```

Comment:
Why use `strtomap` when you could directly create a map?
```suggestion
			tags: map[string]string{"tagkey1": "tagvalue1"}
```